utils/compare.sh

```diff
@@ -2,6 +2,67 @@
 #
 # Handles benchmark-comparison execution
 
+source common.sh
+
+get_network_type() {
+if [[ $NETWORK_TYPE == "OVNKubernetes" ]]; then
+  network_ns=openshift-ovn-kubernetes
+else
+  network_ns=openshift-sdn
+fi
+echo $network_ns
+}
+
+check_metric_to_modify() {
+  export div_by=1
+  declare file_content=$( cat "${config}" )
+  if [[ $file_content =~ "memory" ]]; then
+   export div_by=1048576
+  fi
+  if [[ $file_content =~ "latency" ]]; then
+    export div_by=1000
+  fi
+}
+
+run_benchmark_comparison() {
```

Comment:
The original logic from networking and router tests was different. This is adding the kube-burner's comparison which I think won't work

Reply:
I added a back up way to return the data in utils/csv_modifier.py that I think should cover this. I will run the old way and with my updated code and verify that the data is the same in the sheet.

Reply:
@rsevilla87 (or whoever else is going to review this PR) can you request access to these sheets and I'm only able to give access to everyone with the link which I'm not sure we want that.
I used the same UUID and generated both the below sheets
Old sheet: https://docs.google.com/spreadsheets/d/1Xv1LEJkwk9xnKifRTy7GbmBwNl6Wpkrb8lGRfBBVeq0/edit#gid=1904663657

New sheet (added more data points at bottom): https://docs.google.com/spreadsheets/d/1TAG5Fw4F7uppNgH6UsltwwI8G034CZ5zB2TFqgSF9Bg/edit#gid=2039516611

Any advice on how to order the data in the sheet. That's the one functionality I seem to lose

Reply:
Added environment variable to not sort by the end column value that was reordering the network-perf data

Reply:
ack, thanks
can you add the updated results sheet here if you have one

Reply:
@mohit-sheth newest sheet https://docs.google.com/spreadsheets/d/172Tj5MXbjsSAMEBP4MeEIcmyIDadWnQ9LIGYpNfnWlg/edit?usp=sharing